database/sql/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -100,6 +100,8 @@ class SlickDatabase(override val originalDatabaseConfig: Config) extends SqlData
     actionThreadPool, database.executor.executionContext.reportFailure
   )
 
+  protected[this] lazy val insertBatchSize = databaseConfig.as[Option[Int]]("db.insertBatchSize").getOrElse(2000)
```

Comment:
ToL: Interesting. I only use camelCase for constants. This isn't.
https://www.scala-lang.org/files/archive/spec/2.11/06-expressions.html#constant-expressions